.eslintrc

```diff
@@ -0,0 +1,26 @@
+parserOptions:
+  ecmaVersion: 7
+  sourceType: module
+extends:
+  - eslint:recommended
+  - prettier
+  - plugin:vue/recommended
+  - plugin:import/errors
+  - plugin:import/warnings
+plugins:
+  - import
+  - vue
+settings:
+  import/resolver:
+    # just using node resolver, which gets most of the imports
+    node:
+      extensions:
+        - .js
+        - .vue
+rules:
+  vue/v-bind-style: 2
+  vue/v-on-style: 2
+  vue/html-quotes: [2, 'double']
+  vue/order-in-components: 2
+  # ignores the 'kolibri.*' imports
```

Comment:
You can configure the import plugin to use webpack module resolution by pointing to the webpack config file, but I had no luck with that approach.